<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstXRowsClass</class>
 <widget class="QMainWindow" name="FirstXRowsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FirstXRows</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>80</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Master File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string># of Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Output File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="masterFileLineEdit"/>
     </item>
     <item>
      <widget class="QSpinBox" name="linesSpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputFileLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="masterToolButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>32</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="outputToolButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>32</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="firstxrows.qrc"/>
 </resources>
 <connections/>
</ui>
